src/components/accordion-item/accordion-item.tsx

```diff
@@ -119,48 +125,80 @@ export class AccordionItem {
     });
   }
 
+  disconnectedCallback(): void {
+    disconnectConditionalSlotComponent(this);
+  }
+
+  // --------------------------------------------------------------------------
+  //
+  //  Render Methods
+  //
+  // --------------------------------------------------------------------------
+
+  renderActionsStart(): VNode {
+    const { el } = this;
+    return getSlotted(el, SLOTS.actionsStart) ? (
+      <div class={CSS.actionsStart}>
+        <slot name={SLOTS.actionsStart} />
+      </div>
+    ) : null;
+  }
+
+  renderActionsEnd(): VNode {
+    const { el } = this;
+    return getSlotted(el, SLOTS.actionsEnd) ? (
+      <div class={CSS.actionsEnd}>
+        <slot name={SLOTS.actionsEnd} />
+      </div>
+    ) : null;
+  }
+
   render(): VNode {
     const dir = getElementDir(this.el);
 
-    const iconEl = <calcite-icon class="accordion-item-icon" icon={this.icon} scale="s" />;
+    const iconEl = <calcite-icon class={CSS.icon} icon={this.icon} scale="s" />;
 
     return (
       <Host>
         <div
-          aria-expanded={toAriaBoolean(this.active)}
           class={{
             [`icon-position--${this.iconPosition}`]: true,
             [`icon-type--${this.iconType}`]: true
           }}
-          onClick={this.itemHeaderClickHandler}
-          role="button"
-          tabindex="0"
         >
-          <div class={{ "accordion-item-header": true, [CSS_UTILITY.rtl]: dir === "rtl" }}>
-            {this.icon ? iconEl : null}
-            <div class="accordion-item-header-text">
-              <span class="accordion-item-heading">{this.heading || this.itemTitle}</span>
-              {this.itemSubtitle ? (
```

Comment:
`{this.itemSubtitle ? (` should be `{this.itemSubtitle || this.description ? (` cc @anveshmekala.

Reply:
It wasnt being caught because the storybook still used `itemSubtitle`. Need to make sure we update the storybook as well. Another point for not having two sources of truth here.